django_socio_grpc/tests/fakeapp/grpc/fakeapp.proto

```diff
@@ -251,7 +251,7 @@ message CustomNameForResponse {
 // Test comment for whole message
 message CustomRetrieveResponseSpecialFieldsModelResponse {
     optional string uuid = 1;
-    optional int32 default_method_field = 2;
+    int32 default_method_field = 2;
```

Comment:
is this normal ?

Reply:
yes, default_method_field is a serializer method field with a non optional return type